LocalSettings.php

```diff
@@ -3311,6 +3311,18 @@
 	'wgPopupsOptInDefaultState' => [
 		'default' => 0,
 	],
+	'wgPopupsConflictingNavPopupsGadgetName' => [
+                'default' => '',
+		'gratispaideiawiki' => 'Navigation_popups',
+	],
+	'wgPopupsConflictingRefTooltipsGadgetName' => [
+                'default' => '',
+		'gratispaideiawiki' => 'ReferenceTooltips',
+	],
+	'wgPopupsOptInStateForNewAccounts' => [
+                'default' => null,
```

Comment:
Already the default setting; no need for this (https://github.com/wikimedia/mediawiki-extensions-Popups/blob/master/extension.json#L63)